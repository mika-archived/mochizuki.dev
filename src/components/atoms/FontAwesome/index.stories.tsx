import React from "react";
import { storiesOf } from "@storybook/react";

import FontAwesome from ".";

storiesOf("atoms/FontAwesome", module)
  .add("Brand", () => (
    <>
      <FontAwesome prefix="brands" icon="twitter" />
      Twitter Bird
    </>
  ))
  .add("Solid", () => (
    <>
      <FontAwesome prefix="solid" icon="rabbit" />
      Solid Rabbit
    </>
  ))
  .add("Regular", () => (
    <>
      <FontAwesome prefix="regular" icon="rabbit" />
      Regular Rabbit
    </>
  ))
  .add("Fixed", () => (
    <ul>
      <li>
        <FontAwesome prefix="brands" icon="twitter" fixed />
        Text
      </li>
      <li>
        <FontAwesome prefix="solid" icon="rabbit" fixed />
        Text
      </li>
      <li>
        <FontAwesome prefix="regular" icon="rabbit" fixed />
        Text
      </li>
    </ul>
  ));
