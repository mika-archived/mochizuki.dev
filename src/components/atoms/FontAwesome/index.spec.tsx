import "@testing-library/jest-dom/extend-expect";
import "jest-styled-components";

import React from "react";
import { cleanup, render } from "@testing-library/react";

import FontAwesome from ".";

afterEach(cleanup);

test("does not have changes from snapshot", () => {
  const { asFragment } = render(<FontAwesome prefix="brands" icon="twitter" />);

  expect(asFragment()).toMatchSnapshot();
});
