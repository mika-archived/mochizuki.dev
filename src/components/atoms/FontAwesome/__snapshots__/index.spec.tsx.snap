// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`does not have changes from snapshot 1`] = `
<DocumentFragment>
  <i
    class="fab fa-twitter"
  />
</DocumentFragment>
`;
